### ISSUE TITLE
Redirection is quite bare

### ISSUE THREAD

**Issue by The-EDev** [CrowCpp/crow#86]
Redirection is done through `res.redirect("URL")`, only problem with that is that it requires an absolute URL, meaning the code has to contain the whole `http://domain.xyz/` which is an inconvenience.

However just having routes instead of full URLs isn't as simple as it may seem, requiring the request's URL in order to create the new URL, the problem with that is that it can't be part of the response code, since the response doesn't get any request reference.

In short, 2 solutions seem feasible:
1. Have a local redirect method with a request argument.
2. have the redirect function(s) be separate from the response code, maybe as part of the router code or the app code.

Related to ipkn#346
